src/object/player.cpp

```diff
@@ -311,6 +310,21 @@ Player::do_scripting_controller(const std::string& control_text, bool pressed)
   }
 }
 
+void
+Player::move_to_sector(Sector& other)
```

Comment:
Can `other` be `const Sector&` ?

Reply:
No, because `other` is given as the second argument to `move_object`, which takes a non-`const` reference, because it's modified (the object is moved there).